projects/train/config.yaml

```diff
@@ -49,7 +49,8 @@ data:
     waveform_prob: 0.277
     swap_prob: 0.014
     mute_prob: 0.055
-    trigger_pad: -0.75
+    left_pad: 0.25
+    right_pad: 0.0
```

Comment:
So `pad` refers to how far away from the defining `signal_time` that the respective side of the window will be placed?

Reply:
Exactly, specifically from the whitened kernel. To get the old behavior, you would have `left_pad = right_pad = 0.25`.